### PR TITLE
Added symposium on comp gastronomy event

### PR DESCRIPTION
**Pull request by RudrakshTuwani**
**Event title:** Symposium on Computational Gastronomy

**Organizer's name:** Ganesh Bagler

cc @hasgeek/events-review

### CODE REVIEW

File: _data/events.yml [hunk 1]
```diff
@@ -647,3 +647,12 @@
...
   blurb: Hack on Node.js at HasGeek on 24-25 Feb from 5 pm. Food and snacks on us. Limited seats, sign up now!
+
+- name: symposium-on-computational-gastronomy
+  title: "Symposium on Computational Gastronomy"
+  city: Delhi/NCR
+  venue: IIIT-Delhi	
+  start_time: 2018-03-17 09:00
+  end_time: 2018-03-17 18:30
+  url: https://www.iiitd.ac.in/webstory/gastrosym2018
+  blurb: A one day Symposium on the emerging theme of 'Computational Gastronomy', the data science of food, flavors and health. Conducted by the Complex Systems Laboratory, Center for Computational Biology, IIIT-Delhi. Organizer: Ganesh Bagler
```
Comment (akshaybhalotia) [inline _data/events.yml]: Character `:` is not allowed in between lines. You have 2 options to resolve this:
- replace the `:` with a `-`
- start the line with `blurb: | A one day....`

File: _data/events.yml [hunk 2]
```diff
@@ -647,3 +647,12 @@
...
   url: https://jsfoo.in/2018-feb-hacknight/
   blurb: Hack on Node.js at HasGeek on 24-25 Feb from 5 pm. Food and snacks on us. Limited seats, sign up now!
+  
+- name: symposium-on-computational-gastronomy
+  title: "Symposium on Computational Gastronomy"
+  city: Delhi/NCR
+  venue: IIIT-Delhi
+  start_time: 2018-03-17 09:00
+  end_time: 2018-02-16 19:30
+  url: 2018-03-17 18:30
```
Comment (akshaybhalotia) [inline _data/events.yml]: Put a URL here

File: _data/events.yml [hunk 3]
```diff
@@ -647,3 +647,12 @@
   end_time: 2018-02-25 10:00 
   url: https://jsfoo.in/2018-feb-hacknight/
   blurb: Hack on Node.js at HasGeek on 24-25 Feb from 5 pm. Food and snacks on us. Limited seats, sign up now!
+  
+- name: symposium-on-computational-gastronomy
+  title: "Symposium on Computational Gastronomy"
+  city: Delhi/NCR
+  venue: IIIT-Delhi
+  start_time: 2018-03-17 09:00
+  end_time: 2018-02-16 19:30
```
Comment (akshaybhalotia) [inline _data/events.yml]: End time cannot be before starting time, please check this

**Comment by akshaybhalotia** [comment]
I think this should pass

**Comment by RudrakshTuwani** [comment]
Thank you for your help!

**Comment by akshaybhalotia** [comment]
You're welcome, I'm merging this. You can check https://hasgeek.com in a few minutes.

**Pull request merged**